dspace/config/modules/external-sources/elsevier.cfg

```diff
@@ -1,5 +1,5 @@
 # Api key to be able to make the calls to retrieve the articles, this will need to be requested by the appropriate instance
-external-sources.elsevier.key = ${external-sources.elsevier.key}
+#external-sources.elsevier.key = ${external-sources.elsevier.key}
```

Comment:
This is a circular config reference. I'm not understanding why you'd even have this in a comment, as it's not useful when uncommented. This almost seems to be based on the DSpace 5.x way of managing configs, where config overrides required such odd circular references. In DSpace 6.x (with Apache Commons Config) that is no longer needed, as you can override values in `local.cfg` without these references.  This commented out line should likely have the circular reference removed, as it's not useful.  Or, you could uncomment it again and give it an empty value by default, allowing it to be overridden in `local.cfg`